strategies/failed/pom.xml

```diff
@@ -33,6 +33,11 @@
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.arquillian.smart.testing</groupId>
+      <artifactId>junit-test-result-parser</artifactId>
+      <version>${project.version}</version>
```

Comment:
It shouldn't be here - the whole IDEA of SPI is to make it pluggable, so the "failed" startegy can work for any kind of test result format. This way we make one implementation always available on the classpath and always used.